upup/models/cloudup/resources/addons/networking.weave/v1.9.0.yaml

```diff
@@ -69,3 +79,6 @@ spec:
         - name: cni-conf
           hostPath:
             path: /etc
+        - name: dbus
+          hostPath:
+            path: /var/lib/dbus
```

Comment:
The background to this change is at https://github.com/weaveworks/weave/issues/2427
You can read about the DBus ID at https://dbus.freedesktop.org/doc/dbus-uuidgen.1.html
also https://www.freedesktop.org/software/systemd/man/machine-id.html